### PR TITLE
Create env-vars-setup.cmd

### PR DESCRIPTION
**Pull request by isaac73**
# Description and Motivation 
Create env-vars-setup.cmd
...

# Change type 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: scripts/env-vars-setup.cmd [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+:: -----------------------------------------------------------------------------
+:: DocumentDb Conn String:
+:: {Your Document DB} > Keys > Primary Connection String
+:: Note: use this value for all 3 environment variables below
+::
+:: Example:
+:: PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=AccountEndpoint "https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;"
+:: PCS_TELEMETRY_DOCUMENTDB_CONNSTRING=AccountEndpoint "https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;"
+:: -----------------------------------------------------------------------------
+SETX PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING ""
```
Comment (jillcary) [inline scripts/env-vars-setup.cmd]: We also have:
PCS_STREAMANALYTICS_DOCUMENTDB_CONNSTRING
PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING

Comment (jillcary) [reply scripts/env-vars-setup.cmd]: Are these required?

File: scripts/env-vars-setup.cmd [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+SETX PCS_TELEMETRY_DOCUMENTDB_CONNSTRING ""
+
+:: -----------------------------------------------------------------------------
+:: Auth is disabled for local development
+:: -----------------------------------------------------------------------------
+SETX PCS_AUTH_REQUIRED "false"
+
+:: -----------------------------------------------------------------------------
+:: To add a bing map key, please follow the directions here:
```
Comment (jillcary) [inline scripts/env-vars-setup.cmd]: Change the comment about bing maps as well

File: scripts/env-vars-setup.cmd (hunk 1 above)
Comment (isaac73) [reply scripts/env-vars-setup.cmd]: I don't think so. That's why I didn't add them. We can add them on demand.

File: scripts/env-vars-setup.cmd (hunk 2 above)
Comment (isaac73) [reply scripts/env-vars-setup.cmd]: Do you mean remove these?

File: scripts/env-vars-setup.cmd (hunk 1 above)
Comment (isaac73) [reply scripts/env-vars-setup.cmd]: Are we using PCS_STREAMANALYTICS_DOCUMENTDB_CONNSTRING and PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING. Why are these not there in the main repo ?

File: scripts/env-vars-setup.cmd (hunk 2 above)
Comment (jillcary) [reply scripts/env-vars-setup.cmd]: That would work, or leave documentation on where to find the Azure Maps key.

File: scripts/env-vars-setup.cmd (hunk 1 above)
Comment (jillcary) [reply scripts/env-vars-setup.cmd]: If they aren't in the main repo, then we can leave them out.

Comment (isaac73) [reply scripts/env-vars-setup.cmd]: done

**Pull request merged**